README.md

```diff
@@ -1,2 +1,46 @@
-#Callisto rest api 
+# Callisto REST API
 
+## Running locally
+
+### Create database
+
+If you need to create your own postgres database you can use docker. This works well if you create a container and name it, then you can stop and start it as you please
+
+```sh
+docker create --name postgres-local -e POSTGRES_PASSWORD=Welcome -p 5432:5432 postgres:11.5-alpine
+docker start postgres-local
+```
+
+Connect to database using `psql`
+
+```sh
+psql -h [YOUR DOCKER HOST IP ADDRESS] -U postgres
+```
+
+For example if `echo $DOCKER_HOST` returns `tcp://192.168.64.4:2376` then run `psql -h 192.168.64.4 -U postgres`
+
+Create the database
+```sql
+CREATE DATABASE callisto;
+```
+
+### Setting environment variables 
+
+Before running the database locally, you need to set your local environment variables eg.
+
+```
+DATABASE_ENDPOINT=localhost
+DATABASE_NAME=callisto
+DATABASE_PORT=5432
+DATABASE_USERNAME=postgres
+DATABASE_PASSWORD=Welcome
+```
+
+## Updating the database schema
+
+The following command will deploy the liquibase config using the docker image when run from the root of the project.
+You will need to replace the value [YOUR IP ADDRESS] with the IP address for your database. If you not running on the default port you may also need to update that in the connection string. Check that the username and passsword is also correct
+
+```sh
+docker run -it --rm -v $(pwd)/db/changelog:/liquibase/changelog -v $(pwd)/db/sql:/liquibase/sql liquibase/liquibase  --url="jdbc:postgresql://[YOUR IP ADDRESS]:5432/callisto" --changeLogFile=changelog/db.changelog-main.yml --username=postgres --password=Welcome update
+```
```

Comment:
Love the readme 💚 are there any conventions we need to follow what creating new schema changes that we should detail here? I guess this question is related to my change versioning question above.

Reply:
Yeah good point. Wondering if the changelog naming convention details should be documented in confluence or here. As there's also stuff on confluence about how we structure certain java things too.

Reply:
We agreed that by simplifying the liquibase changelog versioning we remove the need to cover it in the readme